tools/autodeployment/src/sibling-checker/index.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-unused-expressions */
+const fetch = require('node-fetch');
+const { setIntervalAsync, clearIntervalAsync } = require('set-interval-async/dynamic');
+
+const server =
+  process.DEPLOY_TYPE === 'production'
+    ? 'https://telescope.cdot.systems/health.'
+    : 'https://dev.telescope.cdot.systems/health.';
```

Comment:
We could do here what @humphd suggested. We can make the servers check each other, so if the machine is down, we'd still get the status, but I think that you'd need to add a case in which the server never answers.

Reply:
I will sketch this feature. I think it will be better to understand it. But basically, we have the `firstCheck` running every 2 min. If the `firstCheck` fails, it will trigger the `secondCheck`  that stops the `firstCheck` and performs a fetch every 10sec. If the `secondCheck` fails for the first time, the message "SERVER DOWN" is sent to us on SLACK. Once the message is sent, the secondCheck continues to run (the message informing us about the crash is sent just once). When the server is back, `secondCheck` sends the message "SERVER IS BACK", stop itself and triggers the `firstCheck`. 

Because we rely on the network, sometimes the `firstCheck` can emit a false-negative when it fails due to network "issues" (timeout, etc.). That's why I created this system to double-check the server and try to avoid false-negative messages on our slack channel.